Tests/SonarScanner.MSBuild.PreProcessor.Test/CacheProcessorTests.cs

```diff
@@ -181,21 +181,21 @@ public async Task Execute_PullRequest_NoCache()
         [TestMethod]
         public async Task Execute_PullRequest_FullProcessing_NoCacheAvailable()
         {
-            var context = new CacheContext(TestContext, "/k:key-no-cache /d:sonar.pullrequest.base=TARGET_BRANCH");
+            var context = new CacheContext(this, "/k:key-no-cache /d:sonar.pullrequest.base=TARGET_BRANCH");
             await context.Sut.Execute();
 
-            context.Logger.AssertInfoLogged("Processing pull request with base branch 'TARGET_BRANCH'.");
-            context.Logger.AssertInfoLogged("Cache data is not available. Incremental PR analysis is disabled.");
+            logger.AssertInfoLogged("Processing pull request with base branch 'TARGET_BRANCH'.");
+            logger.AssertInfoLogged("Cache data is not available. Incremental PR analysis is disabled.");
             context.Sut.UnchangedFilesPath.Should().BeNull();
         }
 
         [TestMethod]
         public async Task Execute_PullRequest_FullProcessing_WithCache()
         {
-            var context = new CacheContext(TestContext, "/k:key /d:sonar.pullrequest.base=TARGET_BRANCH");
+            var context = new CacheContext(this, "/k:key /d:sonar.pullrequest.base=TARGET_BRANCH");
             await context.Sut.Execute();
 
-            context.Logger.AssertInfoLogged("Processing pull request with base branch 'TARGET_BRANCH'.");
+            logger.AssertInfoLogged("Processing pull request with base branch 'TARGET_BRANCH'.");
```

Comment:
I needed to change this, because there was `logger` and `context.Logger` as a different instance.